stamp_generation/stamp_templates/stamp_base_template.jj2

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html amp lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-fit-text" src="https://cdn.ampproject.org/v0/amp-fit-text-0.1.js"></script>
+    <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+    <script custom-element="amp-twitter" src="https://cdn.ampproject.org/v0/amp-twitter-0.1.js" async></script>
+    <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
+    <script async custom-element="amp-instagram" src="https://cdn.ampproject.org/v0/amp-instagram-0.1.js"></script>
+    <script async custom-element="amp-pinterest" src="https://cdn.ampproject.org/v0/amp-pinterest-0.1.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>{{title}}</title>
+    <link rel="canonical" href="{{ canonical_url }}" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
+    <style amp-custom>
+        amp-story-page:not(.first_page):not(.last_page):before {
+            content:url({{ logo_url }});
+            display:block;z-index:10;position:absolute;top:30px;left:15px;
+            filter:drop-shadow(1px 1px 5px white);
+        }
+
+        @font-face {
+            font-family: 'Roboto', sans-serif;
+            src: url("./fonts/Roboto-Regular.ttf")
+         }
+        .page-title {
+            position: absolute;
+            width: 100%;
+            color: #fff;
+            top: 6%;
```

Comment:
Can everything from style be moved to another css file?

Reply:
We cannot have it as css file but moved it to another template `stamp_css.jj2`

Reply:
Yes, it helps.